bin/common/start_k8s_services.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -o errexit
+mkdir -p /var/lib/kubelet
+mkdir -p /tmp/hostpath_pv
+mkdir -p /home/vagrant/.fissile
```

Comment:
kubelet tries to chown it, but doesn't create it.. so if it isn't created beforehand, kubelet startup fails